extension/content_script.js

```diff
@@ -402,7 +402,10 @@ function toggleKeywordSettings() {
 
 function setupKeyListeners() {
 	let wpmDisplay = $("#speedContainer");
-	jdoc.on("keydown", function(evt) {
+	/* This creates an event namespace so that only these events
+		are removed when off() is called in removeUI. 
+		See: https://api.jquery.com/on/ , section "Event names and namespaces" */
+	jdoc.on("keydown.running", function(evt) {
```

Comment:
Btw can definitely change the name of this namespace

Reply:
very nice usage of the namespace feature!